store.js

```diff
@@ -0,0 +1,66 @@
+const { createStore } = require('./redux.min');
+
+const SELECT = 'select';
+const MOVE_UP = 'move_up';
+const MOVE_DOWN = 'move_down';
+
+const select = (index) => ({ type: SELECT, index });
+const up = () => ({ type: MOVE_UP });
+const down = () => ({ type: MOVE_DOWN });
+
+const initialState = {
+  items: [
+    'Apple',
+    'Bread',
+    'Carrot',
+    'Dumplings',
+    'Eggs',
+    'Fish',
+    'Garlic',
+    'Honey',
+    'Ice cream',
+    'Jam',
+  ],
+  index: null,
+}
+
+const reducer = (state = initialState, action) => {
+  switch(action.type) {
+    case SELECT:
+      return {
+        ...state,
+        index: action.index,
+      };
+    case MOVE_UP:
+      const itemsToMoveUp = [...state.items];
+      [itemsToMoveUp[state.index], itemsToMoveUp[state.index - 1]] = [itemsToMoveUp[state.index + 1], itemsToMoveUp[state.index]]
```

Comment:
лайк за идею

сделаем более читабельным
```suggestion
      [
        itemsToMoveUp[state.index],
        itemsToMoveUp[state.index - 1],
      ] = [
        itemsToMoveUp[state.index + 1],
        itemsToMoveUp[state.index],
      ]
```